core/diagnostics/DiagnosticScenarios/readme.md

```diff
@@ -0,0 +1,32 @@
+# DiagnosticScenarios sample debug target
+
+This sample is for the [diagnostics tutorials](https://docs.microsoft.com/dotnet/core/diagnostics/tutorial/diagnostic-scenarios.md) in the .NET Core Guide.
```

Comment:
The **diagnostics tutorials** URL https://docs.microsoft.com/dotnet/core/diagnostics/tutorial/diagnostic-scenarios.md leads to a 404. Has it not yet been published?

Reply:
Thats correct. It hasn't been published yet. I haven't created the PR yet. It is getting close. What I have so far is on https://github.com/sdmaclea/docs/tree/diagnostioc-scenarios

Reply:
Well, I think the URL isn't correct. the docs.microsoft.com links doesn't include the `.md` extension in the end of URLs.
For example:
The [managed-debuggers.md] on github (https://github.com/sdmaclea/docs/blob/diagnostioc-scenarios/docs/core/diagnostics/managed-debuggers.md) has [This URL](https://docs.microsoft.com/en-us/dotnet/core/diagnostics/managed-debuggers) on docs.microsoft.com, but not [This](https://docs.microsoft.com/en-us/dotnet/core/diagnostics/managed-debuggers.md)